index.js

```diff
@@ -392,10 +394,32 @@ module.exports = {
                       readKeys: [{ key, scheme }],
                       root: lodashGet(msg, 'value.content.root'),
                     }
-                  })
-                )
+                  }),
+                  // aggregate multiple readKeys from invites to different epochs into one invite object
+                  pull.drain(
```

Comment:
this won't be very streamy now, but maybe that's ok for now since we're just getting it to work at first?